redaxo/src/addons/structure/plugins/history/lib/history_login.php

```diff
@@ -33,15 +33,12 @@ public function checkTempSession($historyLogin, $historySession, $historyValidti
     /**
      * @return null|string
      */
-    public static function createSessionKey(#[\SensitiveParameter] $login, $session, $validtime)
+    public static function createSessionKey(#[\SensitiveParameter] $login, $session, $validtime): string
     {
         return password_hash($login . $session . $validtime, PASSWORD_DEFAULT);
```

Comment:
ich denke hier sollten wir eine exception werfen, falls `password_hash()` NULL returned.. es sollte in unserem fall nie NULL returnen - siehe https://github.com/phpstan/phpstan-src/pull/2260

Reply:
ich denke für phpstan genügt ggf ein update: https://github.com/redaxo/redaxo/pull/5635